package.json

```diff
@@ -101,5 +100,10 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "after": "ember-cli-htmlbars"
+  },
+  "packageManager": "pnpm@8.15.9",
+  "volta": {
```

Comment:
We're adding volta?

Reply:
it only matters for folks who have volta installed locally, and affects no one and nothing else

Reply:
I set the tool versions to what CI is using